### DISCUSSION TITLE
🚀 Simple Commerce v6

### DISCUSSION THREAD

**Discussion by duncanmcclean** [duncanmcclean/simple-commerce#1001] [Announcements]
Hey everyone 👋

I'm excited to announce the release of **Simple Commerce v6**. There are a lot of under-the-hood improvements, as well as a few new features and quality-of-life improvements.

I thought it might be nice to highlight a few of the exciting features:

## Digital Products

The Digital Products addon has been merged into the Core Simple Commerce addon. This means it's more closely integrated with other features.

Everything _mostly_ works the same as it did before. However, one different thing, is how you toggle a product to be a digital product.

Instead of the "Is Digital Product" toggle, you'll now see a "Product Type" button group field. Selecting "Digital" will show the relevant digital product fields.

![image](https://github.com/duncanmcclean/simple-commerce/assets/19637309/1e41c0ee-ee96-4df6-b8b9-865de9025897)

For information about other differences, please review the [upgrade guide](https://simple-commerce.duncanmcclean.com/upgrade-guides/v5-x-to-v6-0#content-medium-digital-products-are-now-supported-in-core).

## Dashboard Widgets

Previously, Simple Commerce provided an "Overview" page in the Control Panel, letting your store owners see a chart of sales, a list of recent orders, top customers & products low in stock.

It always felt weird to me that SC had its own widgets page & system so in v6, this "Overview" page has been removed in favour of *real* Dashboard widgets.

Like any other widget, you can configure the displayed widgets in your `config/statamic/cp.php` config file:

```php
'widgets' => [
    [
        'type' => 'orders_chart',
        'width' => 100,
    ],
    [
        'type' => 'recent_orders',
        'limit' => 5,
        'width' => 50,
    ],
    [
        'type' => 'top_customers',
        'limit' => 5,
        'width' => 50,
    ],
    [
        'type' => 'low_stock_products',
        'limit' => 5,
        'width' => 50,
    ],
],
```

Learn more about [CP Widgets in the documentation](https://simple-commerce.duncanmcclean.com/control-panel#content-widgets).

## Using Statamic's users as your customers

By default, all customers are stored as entries. However, if you wish, you can switch them to be [stored as Statamic users](https://simple-commerce.duncanmcclean.com/customers#content-users-driver) or in the [database alongside your orders](https://simple-commerce.duncanmcclean.com/customers#content-database-driver).

Normally, when you're using the `{{ sc:cart:update }}` form, you'd need to provide the user with inputs for their name and email address.

However, often when you are storing customer as Statamic users, your customers will already be logged in so there's not much point in asking them for their details again.

With SC v6, when customers are Statamic users and they're logged in, the customer will automatically be set on their cart, without you (the developer) needing to have any customer-related inputs on the cart forms.

## Other things

* Added a new "Delivered" order status
* The package namespace has changed
* Uses Runway v6 under the hood for database orders
* Products, Customers & Orders now have their own query builders (eg. `Order::query()`)
* Removed the "Enabled" toggle from coupons, in favour of the start/end dates.

For more information on updating, please review the [upgrade guide](https://simple-commerce.duncanmcclean.com/upgrade-guides/v5-x-to-v6-0). You can view the [full changelog here](https://github.com/duncanmcclean/simple-commerce/releases/tag/v6.0.0).

**Comment by Web10-Joris** [comment]
Thank you for the update, Duncan! I'm almost ready to use it in a production-ready environment. Currently, I'm still developing many additional features to test, such as invoices, more advanced stock management, and dynamic tax and shipping rates based on product/weight/dimensions and country.

**Reply by duncanmcclean** [reply]
No worries - hopefully you find some of the new features useful!
